modules/gcve-private-cloud/README.md

```diff
@@ -0,0 +1,100 @@
+# Google Cloud VMWare Engine Private Cloud Module
+
+This module implements the creation and management of a Google Cloud VMWare Engine Private Cloud with its management cluster. If configured, it also creates the VMWare engine network or it can work with an existing one. The creation of the private connection with the user VPC requires the execution of the  [Google SDK command](https://cloud.google.com/sdk/gcloud/reference/vmware/private-connections/create#--routing-mode) the module provides as an output.
+
+To understand the limits and to propertly configure the vSphere/vSAN subnets CIDR range please refer to the [GCVE public documetation](https://cloud.google.com/vmware-engine/docs/quickstart-networking-requirements).
+
+Be aware that the deployment of this module might require up to 2 hours depending on the selected private cloud target zone.
+
+<!-- BEGIN TOC -->
+- [Limitations](#limitations)
+- [Basic Private Cloud Creation](#basic-private-cloud-creation)
+- [Private Cloud Creation with custom nodes and cores count](#private-cloud-creation-with-custom-nodes-and-cores-count)
+- [Variables](#variables)
+- [Outputs](#outputs)
+<!-- END TOC -->
+
+## Limitations
+At the moment this module doesn't support the following use cases:
+- Single node private cloud
+- Stretched private cloud
+
+## Basic Private Cloud Creation
+
+```hcl
+module "gcve-pc" {
+  source          = "./fabric/modules/gcve-private-cloud"
+  name            = "gcve-pc"
+  project_id      = "gcve-test-project"
+  zone            = "asia-southeast1-a"
+  management_cidr = "192.168.0.0/24"
+
+  private_connections = {
+    transit-conn1 = {
+      name              = "transit-conn1",
+      network_self_link = "projects/test-prj-elia-01/global/networks/default",
```

Comment:
we love you but don't put your name in the example. Use a generic name for the project id